scripts/gunCard.js

```diff
@@ -0,0 +1,87 @@
+export class GunCardSheet extends ItemSheet {
+    
+    get template() {
+
+        const path = "systems/vtt-bunkersandbadasses/templates/gunCard.html";
+        if (!game.user.isGM && this.actor.limited) {
+			return path + "limited-sheet.html";
+		}
+        return "systems/vtt-bunkersandbadasses/templates/gunCard.html";
```

Comment:
after merging the start of the path "systems/vtt-bunkersandbadasses" can be replaced by BADASS.systemPath

Reply:
So it could be done before merging ?

Reply:
TODO